src/tests/standings.test.js

```diff
@@ -0,0 +1,13 @@
+import getStandings from './__mocks__/getStandings';
+
+describe('Tests for the standings', () => {
+  test('Get data from the API', () => {
+    expect(getStandings()).toBeDefined();
+  });
+  test('third team name must be Liverpool', () => {
+    expect(getStandings()[2].name).toBe('Liverpool');
+  });
+  test('Points of Barcelona must be 70', () => {
+    expect(getStandings()[3].points).toBe(70);
+  });
+});
```

Comment:
- [ ] None of your components is tested at the moment and no snapshot generated. I would implore you to import your components in your test file and make snapshots which can be done with the use of `toMatchSnapshot()` method provided.